articles/iot-hub/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringV2.md

```diff
@@ -0,0 +1,173 @@
+---
+title: 'IoT DevKit to cloud: Connect IoT DevKit AZ3166 to Azure IoT Hub | Microsoft Docs'
+description: In this tutorial, learn how to send status of sensors on IoT DevKit AZ3166 to Remote Monitoring v2 from Azure IoT Suite for monitoring and visualization.
+services: iot-hub
+documentationcenter: ''
+author: Isabel Cabezas
+manager: 
+tags: ''
+keywords: ''
+
+ms.service: iot-hub
+ms.devlang: arduino
+ms.topic: article
+ms.tgt_pltfrm: na
+ms.workload: na
+ms.date: 12/03/2018
+ms.author: isacabe
+
+---
+# Connect MXChip IoT DevKit AZ3166 to Azure IoT Suite for remote monitoring v2
+
+In this tutorial, you learn how to run a sample app on your DevKit to send sensor data to your Azure IoT Suite.
+
+The [MXChip IoT DevKit](https://aka.ms/iot-devkit) is an all-in-one Arduino compatible board with rich peripherals and sensors. You can develop for it using [Visual Studio Code extension for Arduino](https://aka.ms/arduino). And it comes with a growing [projects catalog](https://microsoft.github.io/azure-iot-developer-kit/docs/projects/) to guide your prototype Internet of Things (IoT) solutions that take advantage of Microsoft Azure services.
+
+## What you need
+
+Finish the [Getting Started Guide](https://docs.microsoft.com/azure/iot-hub/iot-hub-arduino-iot-devkit-az3166-get-started) to:
+
+* Have your DevKit connected to Wi-Fi
+* Prepare the development environment
+
+An active Azure subscription. If you do not have one, you can register via one of these two methods:
+
+* Activate a [free 30-day trial Microsoft Azure account](https://azureinfo.microsoft.com/us-freetrial.html)
+* Claim your [Azure credit](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/) if you are MSDN or Visual Studio subscriber
+
+## Create an Azure IoT Suite
+
+1. Go to [Azure IoT Suite site](https://www.azureiotsuite.com/) and click **Create a new solution**.
+
+  ![Select Azure IoT Suite type](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-v2-solution-types.png)
+ 
+
+2. Select **Remote monitoring preview**.
+

```

Comment:
Would you like to have a notes here, click the 'select' button instead of 'previous version'?